docs/en/mutations.md

```diff
@@ -52,17 +52,17 @@ You can also dispatch mutations using objects:
 
 ``` js
 store.dispatch({
-  type: 'INCREMENT',
-  payload: 10
+  type: 'INCREMENT_BY',
+  payload: [ 10 ]
```

Comment:
Where in the code do you get the impression form that it would be a double/nested array?